README.md

```diff
@@ -69,3 +69,10 @@ yarn deploy
 #### Known issues
 
 - Your project has to run with webpack >= 2. Here's a [guide](https://webpack.js.org/migrate/3/) for upgrading Webpack to v2.
+
+- `VTEX Styleguide` is a project built to run on top of `npm` or `VTEX IO`. To
+  make this viable we currently use two `package.json`, one in the toor repository
+  folder for `npm` and one inside the `react` folder for `VTEX IO`. We are
+  currently studying how to improve this architecture, but for now we have to live
+  with duplicated dependencies in these `package.json` files (For more info please
+  refer to #458, #459 and #483).
```

Comment:
```
`VTEX Styleguide` is a project built to run on top of either `npm` or `VTEX IO`. To
make this viable, we currently use two `package.json` files, one in the top repository
folder for `npm`, and one inside the `react` folder for `VTEX IO`. We are
looking into how we can improve this architecture, but for now we have to live
with duplicated dependencies in these `package.json` files.

For more info, please refer to ....
```

E pro "refer to", cria uma issue só que agrupa essas 3, e coloca o link dela ao invés do numero